--- abyss-1.3.7/configure	2013-12-11 13:38:11.000000000 -0800
+++ patch-files/configure	2014-08-26 11:34:56.471664182 -0700
@@ -7678,7 +7678,8 @@
 fi
 
 # Set compiler flags.
-AM_CXXFLAGS='-Wall -Wextra -Werror'
+# README JJH Stripped -Werror
+AM_CXXFLAGS='-Wall -Wextra'
 
 
 ac_config_files="$ac_config_files Makefile ABYSS/Makefile Align/Makefile Assembly/Makefile Common/Makefile DataLayer/Makefile FMIndex/Makefile Graph/Makefile Parallel/Makefile bin/Makefile doc/Makefile dialign/Makefile kmerprint/Makefile AdjList/Makefile connectpairs/Makefile DAssembler/Makefile DistanceEst/Makefile Layout/Makefile Map/Makefile Misc/Makefile Overlap/Makefile PopBubbles/Makefile Scaffold/Makefile SimpleGraph/Makefile MergePaths/Makefile KAligner/Makefile ParseAligns/Makefile PathOverlap/Makefile Consensus/Makefile FilterGraph/Makefile GapFiller/Makefile Unittest/Makefile"
